Automattic/wp-calypso
# Fix the tooltip styles from the Sensei plan page

Related to #84119

## Proposed Changes

* It fixes the tooltip styles from the Sensei plan page.

**The solution**

I imagine the ideal solution would be to restructure the Sensei onboarding flow to work like the other flows, sharing the same logic, but it seems a very complex task that would take a long time. I also think we'll need to do it at some point anyway. More details in "Considered Alternatives" in https://github.com/Automattic/wp-calypso/pull/77276. cc @Automattic/vertex @Automattic/explorers to see if you have insights about this, since we don't have too much experience with the onboarding yet. Notice that a big difference of Sensei flow is that it's the business plan + Sensei Pro.

As an intermediary solution, I tried to only replace the current tabs component (`PlansIntervalToggle`) with the component the other flows are using (`IntervalTypeToggle`), but it was starting to need some hacks, so I decided to not proceed with it. https://github.com/Automattic/wp-calypso/commit/c8d96f9037c372185253c8f20511d560b1ce003

In the end, I went with the simplest and less invasive solution, but probably not best one. I just tweaked the styles overriding some of them.

## Testing Instructions



* Visit `http://calypso.localhost:3000/setup/sensei/senseiPlan`.
* Change the plan to monthly, and check that the tooltip works properly.
* Visit the plan of other flows (like `http://calypso.localhost:3000/setup/newsletter/plans`), and make sure it's not broken.

## Known issues

* https://github.com/Automattic/wp-calypso/issues/84249
* ~~I wasn't able to access `http://calypso.localhost:3000/setup/sensei/senseiPlan` on Safari because of authentication issues. Not sure if it's something with my environment or a general issue in the development environment. In wordpress.com it works properly though, so I'll make sure I test it on staging before deploying.~~ I was able to test it now. p1700248227094809-slack-C02NWDZBL0H

## Recording

https://github.com/Automattic/wp-calypso/assets/876340/48b7d60f-a93e-4c68-8f00-5bd20e47fcea



## Pre-merge Checklist
